timescaledb/how-to-guides/ingest-data/ingest-kafka.md

```diff
@@ -0,0 +1,23 @@
+# PostgreSQL Kafka connector
```

Comment:
**context:** We have an issue out (blocked while we wait for SME availability) to fix this, because it's pretty confused and unintelligible as-is. Don't think it should block the IA rearrangement, but we'll need to go over this one again later